scripts/rabbitmq-server-ha.ocf

```diff
@@ -260,6 +262,16 @@ ${OCF_RESKEY_binary} should listen on this port
 <content type="boolean" default="${OCF_RESKEY_node_port_default}" />
 </parameter>
 
+<parameter name="vhost" unique="0" required="0">
+<longdesc lang="en">
+Main virtual host used in the cluster, guaranteed to not suffer data loss. Other
+virtual hosts may exist, but some scenarios can lead to data loss for them. This
```

Comment:
What sounds really confusing to me is that all vhosts in RabbitMQ are stored more or less next to each other. Without knowing much what the modern version of this file does, I cannot think of why it would work this way. And I am not a beginner.